### ISSUE TITLE
Mezzanine interface changes

### ISSUE THREAD

**Issue by Spaqin** [sinara-hw/Almazny#7]
There are 11 signals that control Almazny, plus one to read back (``PGOOD``). And they are all around the IO port.
However, Almazny could work with older, 8-pin connector with almost no loss in functionality, while also keeping ARTIQ driver intact.

The suggestion is to change the pinout as follows:
MEZZ_IO0 - SER_MOSI
MEZZ_IO1 - SER_CLK
MEZZ_IO2 - REG_LATCH1
MEZZ_IO3 - REG_LATCH2
MEZZ_IO4 - REG_LATCH3
MEZZ_IO5 - REG_LATCH4
MEZZ_IO6 - REG_OE
MEZZ_IO7 - PGOOD 

REG_OE would a signal that goes to all 4 shift registers (previously REG_NOE1..4), with an inverter in between, to convert it from NOE to OE. Individual RF output can be still adjusted by setting the shift registers individually. The inverter is there to make sure that on powerup the outputs are all disabled, so no unattenuated output is accidentally sent before the shift registers are initialized.

REG_CLEAR (SRCLR#) should be connected to high level at all times. There seems to be no need to clear all the registers at once - that can be done by setting them to 0.

**Comment by gkasprow** [comment]
@jordens how would we detect the mezzanine version? some IOs pulled to GND?

**Comment by jordens** [comment]
IMO since the gateware for the old version is unlikely to ever make it into the "standard mirny gateware", we don't need to distinguish. We can always use the dip switches on mirny to indicate "system configuration", i.e. also mezzanine versions.

**Comment by gkasprow** [comment]
![obraz](https://user-images.githubusercontent.com/4325054/174909912-614e9ed5-1be3-431e-b97d-49a392eb4bc9.png)
![obraz](https://user-images.githubusercontent.com/4325054/174909967-2ecf2f13-4b3d-4563-b6f2-5425e5ddcf2a.png)

**Comment by jordens** [comment]
Ok. That probably didn't work in v1.0. No inversion, no pullup. But keep it.
Still, connect one LED to the unused shift register output QH. Move PGOOD. Add a PGOOD LED, add eeprom and lm75.

**Comment by gkasprow** [comment]
![obraz](https://user-images.githubusercontent.com/4325054/175076212-a05a6898-1481-4f3f-a2ff-a3c0bf325c3c.png)
![obraz](https://user-images.githubusercontent.com/4325054/175076256-df0381af-47ff-4bfb-b52d-591981c83ec7.png)
![obraz](https://user-images.githubusercontent.com/4325054/175076295-fa83adf1-7b71-4cb4-bef1-ababcc715a21.png)
I'm not sure about EEPROM. If they share same address how would you program them?

**Comment by jordens** [comment]
What I meant is:
* PGOOD on MEZZ_IO2, pin 13 as it was in v1.0
* MOSI, CLK, 4xLATCH, NOE on the remaining MEZZ_IO0-7 the way you like them.
* no additional shiftreg
* Use QH output of the four shiftregs to drive four green LEDs one for each channel
* Add PGOOD green front panel LED

**Comment by jordens** [comment]
* They should have different addresses. I thought there were address select pins on those eeproms. Aren't there?

**Comment by jordens** [comment]
Yeah. Then the EEPROM isn't critical. LM75 on a non-colliding address is already useful.

**Comment by gkasprow** [comment]
How to call these additional green LEDs?

**Comment by gkasprow** [comment]
L8...L11 ?

**Comment by jordens** [comment]
Yes.

**Comment by kaolpr** [comment]
As #11 will require modifications, maybe we could reevaluate the lack of EEPROM? 24AA02**5**E48  does have address pins and we could agree on the addressing of mezzanine EEPROMs.

**Comment by gkasprow** [comment]
@kaolpr please check
![obraz](https://user-images.githubusercontent.com/4325054/223847854-f5e04919-1a3c-4555-98ea-73aee214c328.png)

**Comment by gkasprow** [comment]
IC10 is DNP by default

**Comment by kaolpr** [comment]
Unfortunately, this will not work. 

For it to work, we will need to change Mirny EEPROM. Mirny has, as all Sinara modules, 24AA02E48. 
![image](https://user-images.githubusercontent.com/9166701/224037613-59c49bf4-d5dc-4ca0-9465-63a500c06ae4.png)
So no matter how `Ax` lines are set we'll hit Mirny EEPROM. 

However, we can keep it on Mirny but with **DNP jumper** on power supply and move to 24AA02**5**E48 on Mirny in the fitire release.

**Comment by gkasprow** [comment]
let's make the EEPROMs DNP

**Comment by jordens** [comment]
done in v1.2
